assets/airports/airportLoadList.json

```diff
@@ -334,6 +334,12 @@
         "name": "Pittsburgh International Airport",
         "premium": false
     },
+    {
+        "icao": "kpns",
+        "level": "beginner",
```

Comment:
With the currently defined traffic levels, it falls under 'easy'. It must have between 0-20 acph to qualify as 'beginner'